modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -3,28 +3,31 @@
 // * logging/cluster-logging-deploying.adoc
 
 [id="cluster-logging-deploy-multitenant_{context}"]
-= Installing OpenShift Logging into a multitenant network
+= Installing OpenShift Logging with SDN network isolation enabled
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+In a cluster, if the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#nw-openshift-sdn-modes_about-openshift-sdn[OpenShift SDN network isolation mode] uses the _Network policy_ or _multitenant_ modes, projects in that cluster are isolated from each other. Both of these modes prevent network traffic between pods or services that are in different projects. Network policy is the default mode in {product-title} {product-version}.
```

Comment:
@bergerhoffer I reviewed your PR with the updated guidelines for using xrefs. Does this look okay to you?

Reply:
@rolfedh @vikram-redhat @bergerhoffer - did we actually turn on the new `xref` capability yet?

Reply:
Not yet :disappointed: still waiting on the go ahead from @vikram-redhat

Reply:
Removed